README.rst

```diff
@@ -34,6 +34,14 @@ In the Kinto project settings
     kinto.attachment.folder = {bucket_id}/{collection_id}
     kinto.attachment.keep_old_files = true
 
+
+If you want uploaded files to get gzipped when stored:
+
+.. code-block:: ini
+
+    kinto.attachment.gzipped = true
```

Comment:
Yes we can also see it as the default, and add a checkbox gzip files on the kinto-admin so that we pass the querystring.